### ISSUE TITLE
Add support for cryptographic equivocation to support ICS in Gaia v14

### ISSUE THREAD

**Issue by mmulji-ic** [cosmos/relayer#1348]
Gaia v14 introduces an updated ICS provider, which can handle slashing for misbehaving clients in an automated fashion. Therefore Hub governance proposals such as [prop 818](https://www.mintscan.io/cosmos/proposals/818) are no longer required.

Adding this support, to be able to report misbehaviour, would bring the Go relayer in line with [Hermes release 1.7.3](https://github.com/informalsystems/hermes/releases/tag/v1.7.3), which supports this functionality.

**Comment by jtieri** [comment]
Hey @mmulji-ic, getting back to this a little late since it was not in scope of our recent release. 

I'd like to begin scoping this work, but to do so I need to learn a bit more about what supporting this feature entails. Is there somewhere that I can read a spec on cryptographic equivocation?

**Comment by sainoe** [comment]
Hi @jtieri. You can find the cryptographic equivocation [ADR](https://github.com/cosmos/interchain-security/blob/main/docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md.) here.

**Comment by jtieri** [comment]
> Hi @jtieri. You can find the cryptographic equivocation [ADR](https://github.com/cosmos/interchain-security/blob/main/docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md.) here.

thanks! i'm out of the office for the next week but will begin to dig through the ADR asap

**Comment by jonathanpberger** [comment]
<s>Blocked until ICS updates deps.</s>

IDEA: could we downgrade in the relayer for now? Show it works and then build suite after. Then wait on merging until new ICS is ready.

**Comment by jonathanpberger** [comment]
Tried downgrading the relayer, but it was more trouble than it's worth. Iceboxing story.
